app/lib/common/widgets/sliver_scaffold.dart

```diff
@@ -54,12 +53,6 @@ class SideSheet extends StatelessWidget {
       child: Scaffold(
         resizeToAvoidBottomInset: true,
         body: Container(
```

Comment:
we are sure this is not going to bite us later or in some other context that is still using this now?

Reply:
I don't see why this widget still exists in form of ```SideSheet```. The alternative actor of it is ```SideSheetPage``` which helps determining whether UI should open sidesheet or some other kind of dialog. I can't remove this entirely now since its used in many places .i.e. all crud sheets are using this, which would require a lot of refactor. Therefore, I changed it to be a custom sliver widget with app bar. We might need to think on the aspect of determining dialog whether that should on route-level or widget level?